src/main/kotlin/me/sciberras/christian/pvs/ProjectSettingsConfigurable.kt

```diff
@@ -0,0 +1,56 @@
+package me.sciberras.christian.pvs
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.options.Configurable
+import com.intellij.openapi.project.Project
+import com.intellij.ui.dsl.builder.bindItem
+import com.intellij.ui.dsl.builder.panel
+import com.intellij.ui.dsl.builder.toNullableProperty
+import com.intellij.ui.dsl.listCellRenderer.textListCellRenderer
+import javax.swing.JComponent
+
+class ProjectSettingsConfigurable(
+    private val project: Project
+) : Configurable {
+    private val stateService = project.service<ProjectSettings>()
+    private var initialState = this.stateService.state
+    private var currentState = ProjectSettingsState()
+
+    override fun getDisplayName(): String {
+        return "My Project Settings 2" // TODO this seems to be unused
+    }
+
+    override fun createComponent(): JComponent {
+        return panel {
+            row("Refresh language level") {
+                comboBox(
```

Comment:
Might need to store the `comboBox` as a variable and add a listener 👀 
Example: https://github.com/Drewzillawood/intellij-custom-progress-bar/blob/2.1.2/src/main/kotlin/com/drewzillawood/customprogressbar/settings/CustomProgressBarConfigurable.kt#L80-L82